brands/shop/newsagent.json

```diff
@@ -41,6 +41,36 @@
       "shop": "newsagent"
     }
   },
+  "shop/newsagent|Martin's": {
+    "count": 53,
+    "countryCodes": ["gb"],
+    "match": [
+      "shop/convenience|Martin's",
+      "shop/newsagent|McColls"
+    ],
+    "tags": {
+      "brand": "Martin's",
+      "brand:wikidata": "Q7277785",
+      "brand:wikipedia": "en:RS McColl",
+      "name": "Martin's",
+      "shop": "newsagent"
+    }
+  },
+  "shop/newsagent|McColls": {
```

Comment:
According to https://en.wikipedia.org/wiki/McColl%27s this should not be tagged as `shop=convenience` but rather as `shop=newsagent`. So I would consider to remove this entry and leave the other one with the correct match.

Reply:
I think I mixed something up in my last comment. Sorry.
I meant that `McColls` should not be tagged as `shop=newsagent` but rather as `shop=conveniece`. (The other way round as in my first comment)
At least Wikipedia mentions that they are

> trading under the trading nam[e] McColls (for convenience shops)

Reply:
I see what you mean and I changed Martin's to newsagent and McColl's to convenience. When I run npm run build, though, an earlier result of 
`"shop/convenience|Martin's": {
    "count": 53,
    "tags": {
      "brand": "Martin's",
      "name": "Martin's",
      "shop": "convenience"
    }
  },`

keeps coming up in addition to these two entries, even after I delete it. Should I delete it and push as-is, without run build?

Reply:
If you can, you should always run the build script before pushing a commit, because sooner or later somebody else will do this and then there might be a commit with a change that was not intentional...
The problem in your case is that you use the wrong property in the `match` field.
For `shop/newsagent|Martin's` it should be the following:
`"match": ["shop/convenience|Martin's"],`
and for `shop/convenience|McColls` it should be:
`"match": ["shop/newsagent|McColls"],`
Try these instead, delete the old, unwanted entries and then run again the build script.

Reply:
Followed your change requests, then removed the duplicate "McColl's" entry and added "nocount":true to the new "McColls" entry as previously requested. However,  running the build script regenerates the deleted "McColl's" entry and changes "nocount" to "count: 98" in the new "McColls" entry.

Reply:
Is "McColl's" or "McColls" the preferred name?
I'm not sure because the Wikipedia article mentions both names but the current data of OSM indicates that "McColl's" (286 uses) is more used than "McColls" (98 uses)...